implementations/rust/ockam/ockam/src/remote/info.rs

```diff
@@ -0,0 +1,27 @@
+use crate::Message;
+use ockam_core::compat::string::String;
+use ockam_core::{Address, Route};
+use serde::{Deserialize, Serialize};
+
+/// Information about a remotely forwarded worker.
+#[derive(Serialize, Deserialize, Clone, PartialEq, Eq, Debug, Message)]
+pub struct RemoteForwarderInfo {
+    pub(super) forwarding_route: Route,
+    pub(super) remote_address: String,
```

Comment:
I think it makes sense to keep it as String, because it's only used to print it and transfer to the other node. It's not used as an `Address` on the node that operates this struct